src/tools/allocator-resolver.ts

```diff
@@ -23,17 +23,17 @@ import {DatabaseStorageKey} from '../runtime/storageNG/database-storage-key.js';
 import {Handle} from '../runtime/recipe/handle.js';
 import {digest} from '../platform/digest-web.js';
 
-export class StorageKeyRecipeResolverError extends Error {
+export class AllocatorResolverError extends Error {
   constructor(message: string) {
     super(message);
-    this.name = 'StorageKeyRecipeResolverError';
+    this.name = 'AllocatorResolverError';
   }
 }
 
 /**
  * Responsible for resolving recipes with storage keys.
  */
-export class StorageKeyRecipeResolver {
+export class AllocatorResolver {
```

Comment:
I would prefer "AllocatorRecipeResolver" TBH + a new comment explaining the relationship of this class to the Allocator / distributed architecture.

Reply:
I'm happy with that. I'll make the change.